db/schema.rb

```diff
@@ -88,6 +88,19 @@
   add_index "bundle_products", ["bundle_product_id"], name: "fk_bundle_prod_prod", using: :btree
   add_index "bundle_products", ["product_id"], name: "fk_bundle_prod_bundle", using: :btree
 
+  create_table "card_readers", force: :cascade do |t|
+    t.integer  "control_device_id", limit: 4, null: false
+    t.boolean  "entrance",                    null: false
```

Comment:
Hmm, I'm not crazy about this name, but I can't come up with anything better. I originally thought `direction`, but then true/false doesn't really make sense with it. It's probably fine for now and we can change it if we come up with something better.